license.md

```diff
@@ -0,0 +1,26 @@
+This software product is licensed with the BSD 3-clause license.
+
+Copyright (c) 2016, by Youri Arkesteijn, Mitchell Dingjan, Robin van Heukelum, Mathias Meuleman, and Maarten Sijm at Delft University of Technology.
+All rights reserved.
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Delft University of Technology nor the
+      names of its contributors may be used to endorse or promote products
```

Comment:
> Neither the name of Delft University of Technology nor the names of its contributors may be used (...)

It sounds like we are contributors to the DUT instead of our own software product? I'd change it to:

> Neither the name of Delft University of Technology nor the names of _the contributors of this software product_ may be used (...)